dist/main.js

```diff
@@ -0,0 +1,39 @@
+const render = function (roster) {
+    const source = $('#roster-template').html()
+    const template = Handlebars.compile(source)
+    const newHTML = template({roster})
+    $('#team-container').empty().append(newHTML)
+}
+
+//why didn't this work?
+// $('#roster-template').on("error", "img", function() {
+//     $(this).attr('src', 'https://banner2.cleanpng.com/20181026/ajs/kisspng-the-nba-finals-orlando-magic-los-angeles-lakers-at-nba-2-17-free-agents-5bd2d52dc187b0.2626322915405437897927.jpg');
+//   });
+
+const display = function () {
+    const team = $(input).val()
+    $(input).val('')
+    $.get(`/nba/${team}`, function (roster) {
+        render(roster)
+    })
+// }
+
+// // need to do in OOP
```

Comment:
I know this is just you trying to extend the project, but in general we don't like dead (unused) code, even when it is commented out. Later on, we'll learn how to add more features correctly with branches (if you have time you can take a look here --> https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging <-- don't worry if you don't have time, we will learn this next week).